INSTALL.md

```diff
@@ -107,30 +107,57 @@ then `rm -rf vendor/` to remove `client-go`'s copy of its dependencies.
 
 ### Glide
 
-[glide](https://github.com/Masterminds/glide) is another popular dependency
-management tool for go. Glide will manage your /vendor directory, but unlike
-godep, will not modify your $GOPATH (there's no equivalent of `godep restore` or
-`godep save`). To get the `client-go` dependency list into glide's format, you
-can run this series of commands:
+[Glide](https://github.com/Masterminds/glide) is another popular dependency
+management tool for Go. Glide will manage your /vendor directory, but unlike
+godep, will not use or modify your $GOPATH (there's no equivalent of
+`godep restore` or `godep save`).
+
+Generally, it's best to avoid Glide's many subcommands, favoring modifying
+Glide's manifest file (`glide.yaml`) directly, then running
+`glide update --strip-vendor`. First create a `glide.yaml` file at the root of
+your project:
+
+```yaml
+package: ( your project's import path ) # e.g. github.com/foo/bar
+import:
+- package: k8s.io/client-go
+  version: v2.0.0
+```
+
+Second, add a Go file that imports `client-go` somewhere in your project (Glide
+ignores unused imports). Then run the following command in the same directory
+as `glide.yaml` to populate your /vendor directory.
 
 ```sh
-go get k8s.io/client-go
-cd $GOPATH/src/k8s.io/client-go
-git checkout v2.0.0
-# cd 1.5 # only necessary with 1.5 and 1.4 clients.
-glide init # Will import from godep format and produce glide.yaml
+glide update --strip-vendor
 ```
 
-Next there are two possibilities: you could copy the `glide.yaml` file into your
-own project (change the `project: ` line at the top!), or, if you already have a
-`glide.yaml` file, you'll need to merge this `glide.yaml` with it.
+This can also be abbreviated as:
+
+```sh
+glide up -v
+```
+
+Importantly, Glide will detect the versions of dependencies `client-go`
```

Comment:
Could you describe the directory layout after the second step is done? Maybe something like this:

At this point, k8s.io/client-go should be added to your project's vendor/. client-go's dependencies should be flattened and be added to your project's vendor/ as well. 

Glide will detect the versions of dependencies `client-go` specified in client-go's `Godep.json` file, and automatically set the versions of these imports in your /vendor directory. It will also record the detected version of all dependencies in the `glide.lock` file.